ci: ceph fix tests/README.md (#6416)
rook/rook: tests/README.md

To run all integration tests:
```console
go test -v -timeout 1800s github.com/rook/rook/tests/integration
```

> In addition, you can test only one storage provider. For example, you can skip but Ceph tests as follows.

Reviewer: ```suggestion
In addition, you can choose to test only one storage provider. For example, you can run Ceph tests as follows.
```

Author: Fixed, thanks.
